site/en/docs/extensions/mv3/promises/index.md

```diff
@@ -35,11 +37,15 @@ They are an important feature of modern JavaScript, providing benefits such as:
 * Coding in a synchronous style for invoking asynchronous functions
 * A simple "fork and join" syntax for invoking concurrent functions
 
-Extensions can use promises beginning with Manifest V3. Many API methods support promises, and we
-are progressively adding promise support to additional API methods.
+In Manifest V2 extension developers could use libraries to "promisify" Chrome's extensions APIs.
+With the introduction of Manifest V3 many of Chrome's extensions APIs can now return promises. The
+Chrome team is also progressively adding promise support to additional APIs.
 
 {% Aside 'gotchas' %}
-Promises are not available for extensions using Manifest V2, and are not available on all API methods.
+
+Promises are not available for extensions using Manifest V2, and are not available on all API
```

Comment:
This still "[specifically says MV2 extensions cannot use Promises](https://groups.google.com/a/chromium.org/g/chromium-extensions/c/3a3UzLbw1V4/m/FAmdT42UAAAJ)". Something like the following instead?

```suggestion
Promises are not returned by extension APIs under Manifest V2, ...
```

Reply:
More like this:

"and are not yet available on all V3 methods."

Did you mean to put ellipses?

Reply:
Just making a partial suggestion.

Reply:
Since there isn't a single suggestion that I can add to address this feedback, I'll be integrating these comments in a separate commit.